package.json

```diff
@@ -24,50 +39,23 @@
     "react-helmet": "5.2.1",
     "react-icons": "^3.8.0"
   },
-  "keywords": [
-    "gatsby",
-    "tailwind",
-    "tailwindcss",
-    "purgecss"
-  ],
-  "license": "MIT",
-  "scripts": {
-    "analyze:lint": "eslint --ext .jsx --ext .js .",
```

Comment:
what's the reasoning behind removing / changing all existing scripts?

Reply:
Thous s riots was not used. For example this one does the same lint command. Other prittier related scripts also. Prittier now integrated with eslint

Reply:
They are there for convenience. Scripts don't have to be used immediately